assets/src/edit-story/components/form/dropDown/index.js

```diff
@@ -162,7 +164,14 @@ const availableKeysForSearch = [
   '0',
 ];
 
-function DropDown({ options, value, onChange, disabled, ariaLabel }) {
+function DropDown({
+  options,
+  value,
+  onChange,
+  disabled,
+  ariaLabel,
+  lightMode,
```

Comment:
I've come to accept that directly default'ing in the component definition rather than using `defaultProps` is okay.
```suggestion
  lightMode = false,
```